deploy/containers/app.yml

```diff
@@ -0,0 +1,76 @@
+base_image: "rubybench/rubybench_docker"
+
+update_pups: false
+
+params:
+  db_name: rubybench_production
+  db_user: rubybench
+  home: /var/www/ruby-bench-web
+
+templates:
+  - "templates/redis.template.yml"
+  - "templates/postgres.template.yml"
+
+expose:
+  - "3000:3000"
```

Comment:
don't expose 3000, that is internal, expose port 80 NGINX will talk to unicorn on 3000